src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -112,14 +112,14 @@ private void createSymbolicLink() {
 
       // Try to delete the symbolic link in case a dangling link is present
       try {
-        Process process = Runtime.getRuntime().exec("unlink " + hdfsBinaryPath);
+        Process process = new ProcessBuilder("unlink", hdfsBinaryPath).start();
```

Comment:
Please create two lines like this
ProcessBuilder processBuilder = new ProcessBuilder("unlink", hdfsBinaryPath);
Process process = processBuilder.start();

We split this up for clarity and also when there is an error so we can do better error tracking.